### PR TITLE
Documentation refinement

### PR DESCRIPTION
**Pull request by pranita-s**

### CODE REVIEW

File: docs/rdataretriever.rst [hunk 1]
```diff
@@ -25,12 +25,219 @@ To install the development version, use ``devtools``
...
+as shown below:
+
+:: 
+
+  library(rdataretriever)
+  use_RetrieverPath("/home/<system_name>/.local/bin/")
+
+The ``which`` command in the terminal finds the exact location of ``Retriever``, but the path
+required by the function is actually the directory which contains ``Retriever``. Therefore, the
+final folder from the path (*retriever* in this case) needs to be removed before using it.
```
Comment (ethanwhite) [inline docs/rdataretriever.rst]: I would rephrase to:

```
The ``which`` command in the terminal finds the location of ``retriever`` including the name
of the program, but the path required by the function is the directory that contains ``retriever``.
Therefore, the `retriever` needs to be removed from the path before using it.
```

File: docs/rdataretriever.rst [hunk 2]
```diff
@@ -25,12 +25,219 @@ To install the development version, use ``devtools``
...
+when that particular environment is activated. To illustrate, assume the virtual environment is *py27*:
+
+::
+
+  $ conda activate py27
+  (py27) $ which retriever
+  /home/<system_name>/anaconda2/envs/py27/bin/retriever
+
+After the location has been found out, it can be used for ``rdataretriever`` after removing last directory
+from the path as follows:
```
Comment (ethanwhite) [inline docs/rdataretriever.rst]: Rephrase to:

```
This path can be used for ``rdataretriever`` after removing `retriever` as follows:
```

File: docs/rdataretriever.rst [hunk 3]
```diff
@@ -25,12 +25,219 @@ To install the development version, use ``devtools``
...
+**Example** :
+
+::
+
+  rdataretriever :: fetch(dataset = 'portal')
+
+download()
+^^^^^^^^^^
+**Description** : Used to download datasets directly without cleaning them and when user does not 
+have specific preference for format of data and the kind of database.
```
Comment (ethanwhite) [inline docs/rdataretriever.rst]: "have a specific preference for the format of the data..."

File: docs/rdataretriever.rst [hunk 4]
```diff
@@ -25,12 +25,219 @@ To install the development version, use ``devtools``
...
+
+**Example** :
+
+::
+
+  rdataretriever :: get_updates()
+
+reset()
+^^^^^^^
+**Description** : The function will Reset the components of rdataretriever using scope [ all, scripts, data, connection]
```
Comment (ethanwhite) [inline docs/rdataretriever.rst]: We've dropped the storing of connection information in the core retriever. If we're still doing so in `rdataretriever` this is fine. If we've dropped it there as well then we should drop references to `connection`.

File: docs/rdataretriever.rst [hunk 5]
```diff
@@ -56,3 +263,90 @@ To get citation information for the ``rdataretriever`` in R use ``citation(packa
 
 
 .. _Data Retriever: http://data-retriever.org
+
+
+RDataRetriever with PostgreSQL
```
Comment (ethanwhite) [inline docs/rdataretriever.rst]: No capitals in `rdataretriever` here and below`

File: docs/rdataretriever.rst [hunk 6]
```diff
@@ -56,3 +263,90 @@ To get citation information for the ``rdataretriever`` in R use ``citation(packa
...
+==============================
+
+RDataRetriever supports different DBMS for storing of downloaded datasets through the Retriever R API. It supports PostgreSQL, MySQL, SQLite and MS Access.
+
+
+Installation of PostgreSQL in Ubuntu
```
Comment (ethanwhite) [inline docs/rdataretriever.rst]: This is really useful information, but I'm not sure if information on installing postgres into Ubuntu is within scope for our documentation. It seems like if we're going to include this we'd also want to include other DBMSs and OSs. What do you think @pranita-s & @henrykironde?

**Comment by henrykironde** [comment]
@pranita-s I have open a PR pranita-s#2 to do more clean up. Merge it and it will ready for a reviewed.

File: docs/rdataretriever.rst [hunk 7]
```diff
@@ -56,3 +263,90 @@ To get citation information for the ``rdataretriever`` in R use ``citation(packa
...
+
+>>> createdb -U vmsadmin vms;
+
+You will be again prompted for password. After the successful setup of PostgreSQL, it can now be used with R API for Retriever.
+
+Using PostgreSQL with RDataRetriever
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+While using PostgreSQL as a connection type for ``rdataretriever::install`` function, a file named ``postgres.conn`` is needed. It contains information for establishing connection with the requested DBMS, in this case, PostgreSQL. Default location of the file is the directory, through which RStudio is running. If it saved in some other location, its path needs to be given to the install function.
+In the above example, ``postgres.conn`` will look like below:
```
Comment (ethanwhite) [inline docs/rdataretriever.rst]: The core retriever is now handling this through the official `.pgpass` file used by postgres. If this is currently also working in `rdataretriever`, then I think we should remove mention of this old approach in the docs. If it isn't then we can definitely leave this in for now, but it would be good to get `rdataretriever` using `.pgpass` instead.

File: docs/rdataretriever.rst (hunk 6 above)
Comment (pranita-s) [reply docs/rdataretriever.rst]: @ethanwhite @henrykironde I can include information for other DBMSs and OSs too. If there are any content enhancements needed, taking current information for "Installation for PostgreSQL" into consideration, it will be great to know !

File: docs/rdataretriever.rst (hunk 7 above)
Comment (pranita-s) [reply docs/rdataretriever.rst]: Yes, the ``rdataretriever`` is using ``.pgpass`` instead of ``conn`` file. I will make sure, the previous information is edited and also fulfill other requirements.

File: docs/rdataretriever.rst (hunk 6 above)
Comment (ethanwhite) [reply docs/rdataretriever.rst]: I don't think I see anything that needs to be added in general, but I also think it would be find to link to other installation guides and just mention anything that is specific to the retriever that needs to be done.

**Comment by ethanwhite** [comment]
Hey @pranita-s - just wanted to check in on this. It's totally fine to not include installation instructions for all of the different database management systems, so if you have time to do the more minor cleanup then we can go ahead and get this in.

**Comment by ethanwhite** [comment]
Thanks @pranita-s! The changes to `rdataretriever.rst` look great, but it looks like a bunch of other files got accidentally added to this commit as well. If you can clean up and rebase the PR I can get this in. Alternatively you could submit a fresh PR or I can clean things up locally and merge them that way. Just let me know what you'd like to do.

**Comment by pranita-s** [comment]
@ethanwhite I will submit a new PR. Thank you !

**Comment by henrykironde** [comment]
ref #1191
